package.json

```diff
@@ -36,7 +36,7 @@
     "zone.js": "^0.6.23"
   },
   "devDependencies": {
-    "@angular/cli": "^1.0.0",
+    "@angular/cli": "^1.0.2",
```

Comment:
@harshit98 Same line of code. Though there is nothing wrong with it, I think this line needs to be in a relevant PR. We can take a second opinion on this.